languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/words_no_infix_s.txt

```diff
@@ -1594,4 +1594,59 @@ Wirtinnen
 Zacken
 Zeichnerinnen
 Zysten
-Blitzableiter
\ No newline at end of file
+Blitzableiter
+
+Antisemitismus
+Allergiker
+Angehörigen
+Anfragen
```

Comment:
eher "Anfrage", oder? Bsp. Anfragestopp, Anfragesteller...

Reply:
Ich habe in den Daten auch Anfragenbearbeitung, Anfragensysteme, Anfragenummer gefunden? Ich denke, da geht Pl. + Sg.? :-)